dynamic-app/src/main/java/com/okta/idx/android/dynamic/auth/DynamicAuthViewModel.kt

```diff
@@ -324,6 +351,9 @@ internal class DynamicAuthViewModel(private val recoveryToken: String) : ViewMod
         pollingJob = null
     }
 
+    /**
+     * Start aan activity from redirect or proceed to the next step in IDX flow using the current remediation
```

Comment:
This is missing some clarity. "Start an activity from redirect" doesn't give me the context I need to understand this.